tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot

```diff
@@ -54,4 +54,58 @@ Create container without specifying host port
     Should Not Contain  ${output}  Error
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} stop test-redis
     Should Be Equal As Integers  ${rc}  0
-    Should Not Contain  ${output}  Error
\ No newline at end of file
+    Should Not Contain  ${output}  Error
+
+Run and exit with mapped ports
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rm -f $(docker ${params} ps -aq)
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  mkfifo /tmp/fifo1
+    ${result}=  Start Process  docker ${params} run -i --name ctr1 -p 1900:9999 -p 2200:2222 busybox /bin/top < /tmp/fifo1  shell=True  alias=sh1
+    Sleep  5
+    ${rc}  ${output}=  Run And Return Rc And Output  echo q > /tmp/fifo1
+    ${result}=  Wait for process  sh1
+    Log  ${result.stdout}
+    Log  ${result.stderr}
```

Comment:
we need some sort of verification here. at least RC check if nothing else, but preferably something else :)